content/blog/2019/12/2019-12-12-board-election-results.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "2019 Jenkins Board and Officer Elections Results"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- oleg_nenashev
+opengraph:
+  image: /images/post-images/elections-2019/election-2019-opengraph.png
+---
+
+image:/images/post-images/elections-2019/election-2019-opengraph.png[Jenkins Elections, role=center, float=right, height=178]
+
+The Jenkins community is conducting the 2019 elections for Board and Officer positions.
```

Comment:
Since it has happened in the past I would use something like:

"The Jenkins community has recently completed the 2019 elections for Board and Officer positions"